src/XSEPlugin.cpp

```diff
@@ -129,6 +130,7 @@ void MessageHandler(SKSE::MessagingInterface::Message* message)
 					shaderCache.WriteDiskCacheInfo();
 				}
 
+				TruePBR::GetSingleton()->DataLoaded();
```

Comment:
I assume since it's not a true feature we can't rely on the feature data loaded right?

Reply:
Right